met-api/src/met_api/services/comment_service.py

```diff
@@ -0,0 +1,75 @@
+
+"""Service for comment management."""
+from met_api.constants.comment_status import Status
+from met_api.models.comment import Comment
+from met_api.schemas.comment import CommentSchema
+from met_api.schemas.submission import SubmissionSchema
+from met_api.schemas.survey import SurveySchema
+
+
+class CommentService:
+    """Comment management service."""
+
+    otherdateformat = '%Y-%m-%d'
+
+    @staticmethod
+    def get_comment(comment_id, user_id) -> CommentSchema:
+        """Get Comment by the id."""
+        comment = Comment.get_comment(comment_id)
+
+        if comment:
+            if user_id is None and comment.get('status_id', None) != Status.Accepted:
```

Comment:
Also, non authenticated users only have access to comments when the survey/engagement period has been closed

Reply:
working on this

Reply:
done